dev-cpp/asio/asio-1.17.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Asynchronous Network Library"
+HOMEPAGE="https://think-async.com"
```

Comment:
Please also add Github/Sourceforge page here, it seems to be maintained in Github now, not sure if the source tarballs differ for example with docs/examples.